doc/Learn/Maui/MauiOverview.md

```diff
@@ -49,6 +51,28 @@ dotnet new unoapp -preset blank -maui -o SimpleMauiApp
 
 .NET MAUI Embedding can be added to any existing Uno application via the following steps. The .NET MAUI Embedding feature is only supported in Uno applications that target iOS, Android, MacCatalyst, and Windows, as shown in the Platforms folder in the following solution structure. These instructions are appropriate for an Uno application targeting .NET 7.
 
+### MauiEmbedding Dependencies
+
+# [**Single Project**](#tab/single-project)
+
+![Starting solution structure - SingleProject](Assets/GettingStartedExisting01-StartSingleProjectStructure.png)
+
+**UnoFeatures**
+In the class library find the `<UnoFeatures>` property and add `MauiEmbedding`. This step add all the dependencies you need for you to have MauiEmbedding working with you project.
```

Comment:
```suggestion
In the class library find the `<UnoFeatures>` property and add `MauiEmbedding`. This step adds all the dependencies you need for you to have Maui Embedding working with your project.
```

Reply:
Might want to mention something like "in the .csproj" so people know that they need to open that up